src/frontends/tensorflow/src/op_table.cpp

```diff
@@ -421,6 +421,7 @@ const std::map<std::string, CreatorFunction> get_supported_ops() {
         {"AssignVariableOp", CreatorFunction(translate_assignvariable_op)},
         {"AssignAddVariableOp", CreatorFunction(translate_add_variable_op)},
         {"AssignSubVariableOp", CreatorFunction(translate_sub_variable_op)},
+        {"ApproximateEqualOp", CreatorFunction(translate_approximate_equal_op)},
```

Comment:
```suggestion
        {"ApproximateEqual", CreatorFunction(translate_approximate_equal_op)},
```

TensorFlow does not have such operation `ApproximateEqualOp`